model-mesh/base/params.env

```diff
@@ -3,6 +3,6 @@ monitoring-namespace=
 odh-mm-rest-proxy=quay.io/opendatahub/rest-proxy:v0.11.0-alpha
 odh-modelmesh-runtime-adapter=quay.io/opendatahub/modelmesh-runtime-adapter:v0.11.0-alpha
 odh-modelmesh=quay.io/opendatahub/modelmesh:v0.11.0-alpha
-odh-openvino=quay.io/opendatahub/openvino_model_server:2022.3-gpu
+odh-openvino=quay.io/modh/openvino-model-server@sha256:c89f76386bc8b59f0748cf173868e5beef21ac7d2f78dada69089c4d37c44116
```

Comment:
I pushed the image into opendatahub.
Use this quay.io/opendatahub/openvino_model_server@sha256:20dbfbaf53d1afbd47c612d953984238cb0e207972ed544a5ea662c2404f276d